velox/experimental/wave/exec/Instruction.cpp

```diff
@@ -282,7 +282,7 @@ AdvanceResult AbstractReadAggregation::canAdvance(
       deviceStream->hostToDeviceAsync(
           aggState->resultRowPointers->as<int64_t*>() + streamIdx,
           &tempPtr,
-          sizeof(tempPtr));
+          sizeof(*tempPtr));
```

Comment:
Should we say explicitly say `sizeof(int64_t)`?

Reply:
Afaik best practice is to use the variable in case the type changes which likely is rare. I still think this is more clear (understandable) code in the end.

Reply:
I looked at the code again. It looks like we are copying an integer pointer from the host to device?